docs/wiki/framework/medical-treatment-framework.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: wiki
+title: Medical Treatment Framework
+description: Explains extending the treatment system for developers.
+group: framework
+order: 5
+parent: wiki
+mod: ace
+version:
+  major: 3
+  minor: 14
+  patch: 2
+---
+
+## 1. Config Values
+
+### 1.1 Vehicle Patient Seats
+
+Defines the seats that will be prioritized when loading patients into vehicles.
```

Comment:
```suggestion
Defines the seats that will be prioritized when loading patients into vehicles. Uses `moveInCargo` indexes.
```
Not really sure if this is needed, but there are sorta two different numbering systems. The other being for https://community.bistudio.com/wiki/Arma_3:_Actions#GetInCargo